libcodechecker/libhandlers/cmd.py

```diff
@@ -132,23 +132,25 @@ def __add_filtering_arguments(parser):
     Add some common filtering arguments to the given parser.
     """
 
-    # TODO: '-s' does not mean suppressed anymore in any other command.
     parser.add_argument('-s', '--suppressed',
                         dest="suppressed",
                         action='store_true',
-                        help="Show only suppressed results instead of only "
-                             "unsuppressed ones.")
+                        help="DEPRECATED. Show only suppressed results "
```

Comment:
I would mention here that the users should use the `--filter` option to get false positive (suppressed) results.